client/src/util/fetchFromAPI.js

```diff
@@ -0,0 +1,21 @@
+export const queryAPI = (options) => {
+  const options = {
+    url: "http://localhost:5000/menu_items/",
+    onSuccess: (result) => console.log(result.data),
+    onError: (result) => console.log(result.error),
+    method: "GET",
+  };
```

Comment:
This is redeclaring the `options` variable which will cause an error.

I should have been more thorough with my original suggestion.  `options` would come from the calling code and be restructured inside of this function.  Something like this:

```suggestion
  const {url, onSuccess, onError, method} = options;
```